subsys/bluetooth/mesh/adv_ext.c

```diff
@@ -292,13 +292,13 @@ static void send_pending_adv(struct k_work *work)
 		}
 	}
 
-	if (!IS_ENABLED(CONFIG_BT_MESH_GATT_SERVER) ||
-	    !(ext_adv->tags & BT_MESH_ADV_TAG_BIT_PROXY)) {
+	if (IS_ENABLED(CONFIG_BT_MESH_PROXY_SOLICITATION) &&
```

Comment:
Could you elaborate what has been changed after shuffling the sequence of these two conditions? I didn't get.

Reply:
Git weirdly rendered the change. These 2 if statements were reordered.